### PR TITLE
ffmpeg hardware acceleration

### PR DESCRIPTION
**Pull request by USA-RedDragon**
This draft adds support for Intel QSV (QuickSync Video) encoding with ffmpeg into the Docker image.

The Raspberry PI should have some encoding hardware as well ffmpeg might be able to take advantage of, and that's the next goal with this patchset, though I'll need to do some additional work to get it working proper in Docker.

### CODE REVIEW

File: docker/Dockerfile.capture [hunk 1]
```diff
@@ -9,6 +9,19 @@ RUN apt-get update
 RUN apt-get -y upgrade
 
 
+# Intel VA-API drivers
+ARG TARGETARCH
```
Comment (USA-RedDragon) [inline docker/Dockerfile.capture:13]: Not an arg to be passed in, part of `docker buildx` https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/

File: docker/Dockerfile.capture [hunk 2]
```diff
@@ -9,6 +9,19 @@ RUN apt-get update
...
+RUN if [ $TARGETARCH != "arm64" ]; then \
+        sed -i 's/Components: main/Components: main non-free non-free-firmware/g' /etc/apt/sources.list.d/debian.sources \
+        && apt-get update \
+        && apt-get -y install \
+        --no-install-recommends \
+        --no-install-suggests \
+            intel-media-va-driver-non-free \
```
Comment (USA-RedDragon) [inline docker/Dockerfile.capture:20]: Non-free drivers required to _encode_ video with QSV

Comment (USA-RedDragon) [reply docker/Dockerfile.capture:20]: 9d5ae2007322ae0c55bcb1fc9db846baeaa3d558

**Comment by aaronwmorris** [comment]
I am ready to merge this if you are.

**Comment by USA-RedDragon** [comment]
Yep, this patchset's been working for me in my Kubernetes cluster for a couple days now.

**Pull request merged**